### PR TITLE
Expand API docs frontpage

### LINKED ISSUE #727: Internal API endpoints are documented in API reference

### PR DESCRIPTION
**Pull request by stefanv**
- Add API access description
- Upgrade Redoc
- Remove API docs for /api/internal targets
- Add SkyPortal logo and link through logo to docs

Closes #727

### CODE REVIEW

**Comment by dannygoldstein** [comment]
Can I go ahead and merge this?

**Comment by stefanv** [comment]
I'm done with it for now.

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "react-simple-dropdown": "3.2.3",
     "react-window": "1.3.1",
     "redoc": "2.0.0-rc.31",
-    "redoc-cli": "0.9.8",
+    "redoc-cli": "0.9.11",
```
Comment (dannygoldstein) [inline package.json:42]: Is this the change that is causing GA to fail?

Comment (stefanv) [reply package.json:42]: I think it's a co-depdency of redoc itself.  I had it at redoc rc.37 which broke things.  Downgrading didn't help.  We can try and pull this version down too.  But why isn't Travis-CI complaining?

**Pull request merged**